src/components/Category.js

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { statusCheck } from '../redux/categories/categories';
 
-const Category = () => (
-  <button type="button">Check Status</button>
-);
+const Category = () => {
+  const dispatch = useDispatch();
+
+  const handleClick = () => {
+    dispatch(statusCheck());
+  };
+
+  return (
+    <>
+      <div>
+        <button type="button" onClick={handleClick}>Check Status</button>
```

Comment:
- [ ] Kindly make sure that the `check status` button on click shows under construction, please make sure the function created dispatches the action created 👍🏿